keyform/templates/keyform/emails/status_update.html

```diff
@@ -0,0 +1,422 @@
+<!doctype html>
+{% load i18n %}
+<html>
+
+<head>
+    <meta name="viewport" content="width=device-width" />
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <title>{% trans 'Simple Transactional Email' %}</title>
```

Comment:
This was just something in the HTML template for the email that I got. I can take it out.